packages/macos/generate_wazuh_packages.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -ex
+set -e
```

Comment:
For this script, we already have logic in place that allows setting the --verbose flag, which enables set -x.